spk/plexpy-custom/src/config.ini

```diff
@@ -0,0 +1,3 @@
+[General]
+launch_browser = 0
+web_port = 8183
```

Comment:
Have changed to 8010 if I understand correctly, although I see that most of the other packages (1 or odd numbers for http), don't follow the convention (perhaps came after). PlexPy default port is 8181